### PR TITLE
Chore documentation

### PR DESCRIPTION
**Pull request by galenwinsor**

### CODE REVIEW

File: src/trials/welcome.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { lang } from '../config/main'
 import { photodiodeGhostBox } from '../lib/markup/photodiode'
 import { baseStimulus } from '../lib/markup/stimuli'
 
-const pleaseBiggen = () => {
+const pleaseBegin = () => {
```
Comment (mcmcgrath13) [inline src/trials/welcome.js]: this one isn't a typo (though perhaps not well named either) - it's asking the user to make the window larger

**Review by mcmcgrath13 — CHANGES_REQUESTED** [review]

File: src/trials/welcome.js (hunk 1 above)
Comment (galenwinsor) [reply src/trials/welcome.js]: Haha, I see. Fixed!

**Review by mcmcgrath13 — APPROVED** [review]
🆎

**Comment by RobertGemmaJr** [comment]
@digicosmos86 @mirestrepo Do you think we should merge this in as well? I think the documentation would be really nice but to have but I don't know enough to know if it's out of date at this point

Actually it's base is #88 so I say we merge it there and take a deeper look in that branch?

**Comment by mirestrepo** [comment]
> @digicosmos86 @mirestrepo Do you think we should merge this in as well? I think the documentation would be really nice but to have but I don't know enough to know if it's out of date at this point
> 
> Actually it's base is #88 so I say we merge it there and take a deeper look in that branch?

Sounds like a plan!

**Pull request merged**